conans/client/command.py

```diff
@@ -311,7 +311,8 @@ def info(self, *args):
                             'e.g., MyPackage/1.2@user/channel or ./my_project/')
         parser.add_argument("--file", "-f", help="specify conanfile filename")
         parser.add_argument("--only", "-n", nargs="?", const="None",
```

Comment:
Missing: update, required, date, author.
And for --path option: export_folder, build_folder, package_folder, source_folder